app/_components/stations/types.ts

```diff
@@ -0,0 +1,28 @@
+export type StationForm = {
+  stationId?: string;
+  ships?: {
+    value: string;
+  }[];
+  modules?: {
```

Comment:
I had started a types folder on project root, but I am good with keeping component specific types within the component folders if they aren't useful outside

Reply:
This is my preferred approach but I think the commodity type was in a subfolder so I just did the same.

I've moved all the form types into the `_types` dir